lab-brae/test/greet-test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const greet = require('../lib/greet.js');
+const expect = require('chai').expect;
+
+describe('Greet Module', function() {
+  describe('#sayHello', function() {
+    it('should return Hello brae', function() {
+      var result = greet.sayHello('brae');
+      expect(greet).to.have.property('sayHello');
+      expect(result).to.equal('Hello brae');
```

Comment:
Nice work storing the result of the method call above into a variable to test on. Helps the test look a lot cleaner.